invokeai/app/services/config/invokeai_config.py

```diff
@@ -323,6 +325,14 @@ def get_config(cls, **kwargs) -> InvokeAIAppConfig:
             cls.singleton_init = kwargs
         return cls.singleton_config
 
+    @classmethod
+    def clear_config(cls):
+        """
+        This removes the singleton InvokeAIAppConfig configuration object.
+        """
+        cls.singleton_config = None
+        cls.singleton_config = None
+
```

Comment:
I don't see this called anywhere

Reply:
Removed. Was a workaround for a problem that I fixed in a better way.